chapters/annotations.tex

```diff
@@ -39,14 +39,19 @@ \section{Documentation}\label{annotations-for-documentation}\label{documentation
 The \fmtannotationindex{Documentation} annotation has the following contents, where the \lstinline!info! and \lstinline!revisions! annotations are described in \cref{annotation-info-revisions}, and the \lstinline!figures! annotation is described in \cref{annotations-for-figures}:
 \begin{lstlisting}[language=modelica]
 record Documentation
-  String info = "" "Description of the class";
-  String revisions = "" "Revision history";
+  constant String info = "" "Description of the class";
+  constant String revisions = "" "Revision history";
   Figure[:] figures = {}; "Simulation result figures";
 end Documentation;
 \end{lstlisting}
 
 How the tool interprets the information in \lstinline!Documentation! is unspecified.
 
+\subsection{How to interpret annotation definitions}
+If the annotation itself is described by a \lstinline!record! \emph{class} the annotation can be seen as a modifier for a \lstinline!record! \emph{component} with the same name.
+If the annotation is declared as a parameter the corresponding modifier must be an evaluable parameter.
```

Comment:
To avoid confusion, couldn't we write something more than just `parameter` when it is evaluable parameter we mean?  For instance:
```suggestion
If the annotation is declared as a parameter the corresponding modifier must be a parameter, or an evaluable parameter in case the \lstinline!parameter! keyword is prefixed with $\mathit{evaluable}$, as in \lstinline!$\mathit{evaluable}$ parameter Boolean HideResult!.
```

Reply:
But we are trying to use the definitions as they exist in language now, and _evaluable_ is not an official keyword. I prefer Hans's formulation. Same for constants below.

Reply:
I also prefer to stick with what is available now, even if slightly inaccurate.

Reply:
This has now been addressed.  Resolving.